srcpkgs/cinnamon-screensaver/template

```diff
@@ -16,7 +16,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="https://github.com/linuxmint/cinnamon-screensaver"
 distfiles="https://github.com/linuxmint/${pkgname}/archive/${version}.tar.gz"
-checksum=0e0dff969e5fbfa3c81912f336ed43b23f69b1277f34c1108ad5d7d8a38787a7
+checksum=fdc141a4769c1ceb70d672269aed748df323c38e1a05e03d7c586123de327530
 make_check=no
```

Comment:
Could you describe the failure?

Reply:
like that?

Reply:
Then it could be fixable with xvfb-run.

Reply:
weird... I might have confused this with another package which I tried to enable tests. This one doesn't generate any tests.